.circleci/config.yml

```diff
@@ -259,7 +259,7 @@ jobs:
       - attach_workspace:
           at: ~/
       - browser-tools/install-chrome
-      - run: yarn run test-render
+      - run: CI=true yarn run test-render
```

Comment:
CircleCI always sets the `CI` env to `true`, so this should not be necessary https://circleci.com/docs/env-vars#built-in-environment-variables

Reply:
Just tested this with ssh-ing to circleci - and confirmed CI does not need to be explicitly set. Thanks Stepan!